src/app/store.js

```diff
@@ -0,0 +1,6 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
You should create redux folder

Reply:
@

> You should create redux folder

@ba-lambert I have already created folder called "reducers" in `src` directory, which will hold all Redux Slices.

Reply:
But also the folder should contain all actions plus stores that why you have to call it redux 😡

Reply:
I think me also u can create a single folder which will hold all activities related to redux like store, reducers (slice), actions. to avoid disorder of folders

Reply:
you're not following the redux folder stucture as they said above you need to have the redux folder and inside that folder you can have another folders like store,actions,constants and others

Reply:
I have renamed reducers to redux directory that hold every thing related to redux. I want to let you know that I have used latest technology of Redux toolkit where we don't need to have separate reducers and actions. I would like if all of us use createSlice function from Redux toolkit. Includes utilities to simplify common use cases like store setup, creating reducers, immutable update logic, and more. We can do more work with less code.